### PR TITLE
Handle ordered lists as well

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Part of #40912.

r? @rust-lang/docs

### CODE REVIEW

File: src/librustdoc/html/markdown.rs [hunk 1]
```diff
@@ -427,12 +427,15 @@ pub fn render(w: &mut fmt::Formatter,
...
+            if shorter.is_compact() {
+                break
+            }
         }
         buffer.push_str(&format!("<li>{}</li>", content));
     }
 
     fn list(parser: &mut ParserWrapper, buffer: &mut String, toc_builder: &mut Option<TocBuilder>,
-            shorter: MarkdownOutputStyle) {
+            shorter: MarkdownOutputStyle, is_sorted_list: bool) {
```
Comment (steveklabnik) [inline src/librustdoc/html/markdown.rs]: shouldn't this be "ordered" and not "sorted"

**Review by steveklabnik — COMMENTED** [review]
one tiny name change, then this looks good to me

File: src/librustdoc/html/markdown.rs (hunk 1 above)
Comment (steveklabnik) [inline src/librustdoc/html/markdown.rs]: I don't think this is a huge deal

Comment (GuillaumeGomez) [inline src/librustdoc/html/markdown.rs]: Oups, good point!

**Comment by frewsxcv** [comment]
Note that if the rollup lands, your commit is not going to be included

**Comment by steveklabnik** [comment]
Yes, i should have just not commented.

**Comment by GuillaumeGomez** [comment]
Ah damn. :'(

I'll write a "fix" afterwards then.

**Comment by steveklabnik** [comment]
This was actually merged, the amendment after didn't make it, but that's fine.

**Pull request closed without merging**